Smartsheet.Net.Standard/Http/SmartsheetHttpClient.cs

```diff
@@ -1101,5 +1101,14 @@ private void InitializeNewRequest()
             return response.Result;
         }
         #endregion
+    
+        #region Cross Sheet Refs 
+
+        public async Task<IEnumerable<CrossSheetReference>> ListCrossSheetReferences(long sheetId, string accessToken = null) {
+            var response = await this.ExecuteRequest<IndexResultResponse<CrossSheetReference>, User>(HttpVerb.GET, string.Format("sheets/{0}/crosssheetreferences", sheetId), null, accessToken: accessToken);
```

Comment:
Add the following before calling ExecuteRequest:

if (sheetId == null)
{
    throw new Exception("Sheet ID cannot be null");
}